lib/tasks/instrumentation_generator/instrumentation.thor

```diff
@@ -85,18 +87,35 @@ class Instrumentation < Thor
     )
   end
 
-  def config_block(library)
+  def append_to_newrelic_yml(name)
+    insert_into_file(
+      NEWRELIC_YML_LOCATION,
+      yaml_block(name),
+      after: "# instrumentation.bunny: auto\n"
+    )
+  end
+
+  def config_block(name)
     <<-CONFIG
-        :'instrumentation.#{library}' => {
+        :'instrumentation.#{name.downcase}' => {
           :default => 'auto',
           :public => true,
           :type => String,
           :dynamic_name => true,
           :allowed_from_server => false,
-          :description => 'Controls auto-instrumentation of the #{library} library at start up. May be one of [auto|prepend|chain|disabled].'
+          :description => 'Controls auto-instrumentation of the #{name} library at start up. May be one of [auto|prepend|chain|disabled].'
         },
     CONFIG
   end
+
+  def yaml_block(name)
+<<HEREDOC # rubocop:disable Layout/IndentationWidth
```

Comment:
Cop disabling required to get the appropriate indentation in the .yml file

Reply:
The squiggle heredoc format would be perfect here:

```ruby
  def yaml_block(name)
    <<~INSTRUMENTATION_CONFIG_INFO
      # Controls auto-instrumentation of #{name} at start up.
      # May be one of [auto|prepend|chain|disabled]
      # instrumentation.#{name.downcase}: auto
    INSTRUMENTATION_CONFIG_INFO
  end
```

It requires Ruby 2.3+. If this generator is intended to support Ruby 2.2, we could leave a TODO to suggest moving to a squiggle heredoc once the option is available.